backend/kesaseteli/applications/models.py

```diff
@@ -95,12 +95,10 @@ class SummerVoucher(HistoricalModel, UUIDModel):
         blank=True,
         verbose_name=_("employee school"),
     )
-    employee_ssn = encrypt(
-        models.CharField(
-            max_length=32,
-            blank=True,
-            verbose_name=_("employee social security number"),
-        )
+    employee_ssn = EncryptedCharField(
+        max_length=32,
+        blank=True,
+        verbose_name=_("employee social security number"),
```

Comment:
If you need to enable "exact" queries by employee_ssn, do you also need to add a SearchField to the model in conjunction with the EncryptedField?

Reply:
Good point! As far as I know, we won't need to query the SSN field though.